MusicNotationKit/MusicNotationKit/Tuplet.swift

```diff
@@ -60,6 +74,7 @@ public struct Tuplet: NoteCollection {
         guard count > 1 else {
             throw TupletError.countMustBeLargerThan1
         }
+        noteCount = count
```

Comment:
If `noteCount = count`, then it is probably better to keep `noteCount:Int { return notes.count}` or something like that?  I am just concerned about cases in which we replace a note with a couple of notes, and the note count needs to be updated as a result?  I am not sure if I am following correctly.